build/sync/plan.yml

```diff
@@ -0,0 +1,30 @@
+checks:
+  - type: repo_is_clean
+    params:
+      repo: template
+  - type: repo_is_clean
+    params:
+      repo: plugin
```

Comment:
I'm 1/5 that it's not worth adding `plan.yml` at this point. Once https://mattermost.atlassian.net/browse/MM-28076 is resolved, we can use that.